sources/api/host-containers/src/main.rs

```diff
@@ -105,9 +105,9 @@ mod error {
         #[snafu(display("Logger setup error: {}", source))]
         Logger { source: log::SetLoggerError },
 
-        #[snafu(display("Unable to base64 decode user-data '{}': '{}'", base64_string, source))]
+        #[snafu(display("Unable to base64 decode user-data for container '{}': '{}'", name, source))]
         Base64Decode {
-            base64_string: String,
+            name: String,
             source: base64::DecodeError,
         },
```

Comment:
Do we want to make the same change to the `Base64Decode` error in `bootstrap-containers`?

Reply:
bootstrap-containers doesn't include the string, but I'll make a similar change in all the places we have a Base64Decode and post a new revision.